saas/{{cookiecutter.project_name}}/webapp/src/routes/+page.ts

```diff
@@ -0,0 +1,6 @@
+import { redirect } from '@sveltejs/kit';
+import { verifyUser } from '$lib/auth/verifyUser';
+
+export const load = async () => {
+	if (await verifyUser()) throw redirect(303, '/my');
+};
```

Comment:
all this stuff, does work also when the auth feature flag is off?

Reply:
yup